pkg/operator/ceph/file/mds/spec.go

```diff
@@ -35,6 +35,17 @@ const (
 	// MDS uses approximately 125% of the value of mds_cache_memory_limit in RAM.
 	// Eventually we will tune this automatically: http://tracker.ceph.com/issues/36663
 	mdsCacheMemoryLimitFactor = 0.5
+
+	// NOTE:
+	// Prior k8s version v.1.17 the label to use for labeling zones in nodes is:
+	//		v1.LabelZoneFailureDomain       = "failure-domain.beta.kubernetes.io/zone"
+	// Starting with k8s v.1.17 the previous label has been deprecated in favour of
+	//		v1.LabelZoneFailureDomainStable = "topology.kubernetes.io/zone"
+	//
+	// The right constant won't be present in Rook until we will update the k8s library
+	// So the following local constant is used to provide the right label caption,
+	// until this moment arrives:
+	labelZoneFailureDomain = "topology.kubernetes.io/zone"
```

Comment:
If they are running on 1.16 or older, we should use the older label, right? Looks like you are always using the newer label.

Reply:
I think this value is used to find the topology label key applied by users, which means that this code has to look for both the old and new labels since the old ones are supported for backward compatibility until like k8s v1.20 or something. But this is complexity we wouldn't have to consider if we scheduled using canaries.